samples/python/bulk_edit_issues.py

```diff
@@ -0,0 +1,47 @@
+import requests
+import json
+
+page_size = 1000 #max value
+page = 0
+api_key = "<api-key>"
+endpoint = "https://api.helpshift.com/v1/<domain>/issues"
+
+def bulk_edit_issue(issues):
+    print ("Triggering Bulk Action for " + str(len(issues)) + " issues")
+    api_params = {"issue-ids": json.dumps([issue['id'] for issue in issues]),
```

Comment:
`api_params` in this case (1000 issues) can result in the server returning `413 Request Entity Too Large` Error.

Reply:
Why? The same APIs are used for more than 5k issues.

Reply:
Resolved